DancingGoat/Utils/ProjectUtils.cs

```diff
@@ -1,11 +1,24 @@
 ﻿using System;
+using System.Configuration;
+using System.Linq;
+using System.Web;
 
 namespace DancingGoat.Utils
 {
     public class ProjectUtils
     {
+        private static string AppSettingPreviewApiKey = "PreviewApiKey";
+        private static string PreviewApiKeyCookieName = "PreviewApiKey";
+
         public static string GetProjectId()
         {
+            // check web.config first
+            if (Guid.TryParse(ConfigurationManager.AppSettings["ProjectId"], out Guid projectId))

```

Comment:
I think this block of code should be only in master.

Reply:
I think so as well, we should consider it after closing this task.